Support adding plugin engine / file manager / terminal through plugins (#254)
oeway/ImJoy: README.md

of the local GPU/CPU. The computational backend can also be launched on remote servers
including cloud servers on Amazon, Google Compute, or an institutional computing cluster.

> Previously, running Python plugins will require the installation of Plugin Engine. However, now we have added an experimental support for running Python plugin via the servers provided by MyBinder.org.

Reviewer: ```suggestion
Previously, running Python plugins would require installation of the Plugin Engine. However, now we have added experimental support for running Python plugins via the servers provided by MyBinder.org.
```

> So you an try for example [DeepBindScan](https://imjoy.io/#/app?plugin=oeway/ImJoy-Plugins:DeepBindScan) without any installation.

Reviewer: ```suggestion
You can try for example the [DeepBindScan](https://imjoy.io/#/app?plugin=oeway/ImJoy-Plugins:DeepBindScan) plugin without any installation.
```
